.github/workflows/create-release-pr.yml

```diff
@@ -0,0 +1,19 @@
+name: create-release-pr
+
+on:
+  schedule:
+    # Every monday at 4pm UTC (9am PST)
+    - cron: '0 16 * * 1'
+    
+
+jobs:
+  create-release-pr:
+    runs-on: ubuntu-latest
+    steps:
+    # checks out out repo to $GITHUB_WORKSPACE
```

Comment:
Typo in comment, I'm guessing you might have meant checks out the repo?

Reply:
I don't know what I meant. I copy-pasted this 😄 I will fix.

Reply:
Fixed!